synthnn/core/accelerated_musical_network.py

```
@@ -330,6 +350,14 @@ def batch_process_signals(self, signals: List[np.ndarray],
         Returns:
             List of results
         """
+        # Support legacy call where sample_rate was passed instead of operation
+        if isinstance(operation, (int, float)):
+            warnings.warn(
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)